Firmware/RTK_Surveyor/Display.ino

```diff
@@ -512,13 +515,17 @@ void displaySplash()
     {
       printTextCenter("Facet LB", yPos, QW_FONT_8X16, 1, false);
     }
+    else if (productVariant == REFERENCE_STATION)
+    {
+      printTextCenter("Ref Stn", yPos, QW_FONT_8X16, 1, false);
```

Comment:
Ref Stn not CORS? Your choice.

Reply:
The PCB silk, labels etc. all now say "Reference Station". I know what CORS means, but some users might not. Ref Stn fits the line width and is - I think - a bit more universal and easily understood?

Reply:
Did they know what RTK meant before they started playing with these products? Keep Ref Stn. It's good.